src/components/m-table-edit-cell.js

```diff
@@ -9,6 +9,10 @@ class MTableEditCell extends React.Component {
     super(props);
 
     this.state = {
+      errorState: {
```

Comment:
Is errorState a material-ui prop, or is it a new @material-table/core prop?

Thanks for this! I did some research on errorState prop and couldn't find anything in the material-ui docs. (sorry if it's a dumb question).